### ISSUE TITLE
Custom color for points based on attribute not working

### ISSUE THREAD

**Issue by ScubaDiving** [potree/potree#1488]
I'm trying to view a point cloud I created using COLMAP and then modified using `laspy` to have a new custom attribute called `building_id`.

I then created a las file from the txt point cloud and converted the laspy using potreeconverter 2.0.

When viewing in potree, the points are not colored according to the new attribute even though it seems the values are correctly displayed (should be 1 to about 260).

When I move the slider however, the points do not start dissappearing according to the selected area.

What is going wrong?

I uploaded the las file, **remove the .json filetype, it is las**
[points3D_modified.las.json](https://github.com/user-attachments/files/18112242/points3D_modified.las.json)

I used this conversion command:

```
PotreeConverter/build/Release/potreeconverter.exe -i PotreeConverter/build/points3D_modified.las -o potree/pointclouds/potree_format --generate-page new_pointcloud_page
```

Here's how it looks:

![image](https://github.com/user-attachments/assets/6522843b-cb9e-4cf6-9e34-1670ef27b3ff)

**Comment by GEORGIAN86** [comment]
How you added your Custom Atrribute in Potree ?

**Comment by ScubaDiving** [comment]
> How you added your Custom Atrribute in Potree ?

I'm not sure what do you mean.

I added the custom attribute in the LAS file using a python script I wrote with laspy. I then converted to potree with potreeconverter 2.0 with no specific attribute parameters (so all of them, I think). 
When I select the point cloud in potree, it allows me to change the coloring attribute (see image), but it has no effect on the colors when I choose my custom attribute.

Since other attributes work, I assume it's due to the large number of colors required (260)? is there a limit (8bit?) this number is over?

**Comment by lymperis-e** [comment]
Any updates on this? Facing the same issue using a uint8 custom field

**Comment by ScubaDiving** [comment]
> Any updates on this? Facing the same issue using a uint8 custom field

Not at all. I don't know what to do to solve this. I found some ugly workarounds (as in, to export chunks of the point cloud per each category), but nothing for this visualization.

**Comment by Dingo61** [comment]
please see Issue #1085 for a fix in PotreeRenderer.js

**Comment by spadarian** [comment]
Thanks for this amazing project!

Did you manage to solve your problem @ScubaDiving ?

I'm facing the same problem when adding a custom attribute. I'm not super familiar with the code base but I think the problem is not renderer.

Checking the converter ([`chunker_countsort_laszip.cpp:588`](https://github.com/potree/PotreeConverter/blob/c2328c433c7776e231d86712bb4074c82659e366/Converter/src/chunker_countsort_laszip.cpp#L588)), the `src` and `dst`values are different from `0`.

Having a look at [`DecoderWorker.js:134`](https://github.com/potree/potree/blob/5636cd471d9eb464969e758be45c44d7613d3859/src/modules/loader/2.0/DecoderWorker.js#L134), the value of the custom attribute is always `0`.

Happy to help with extra debugging!

**Comment by ScubaDiving** [comment]
> Thanks for this amazing project!
> 
> Did you manage to solve your problem [@ScubaDiving](https://github.com/ScubaDiving) ?
> 
> I'm facing the same problem when adding a custom attribute. I'm not super familiar with the code base but I think the problem is not renderer.
> 
> Checking the converter ([`chunker_countsort_laszip.cpp:588`](https://github.com/potree/PotreeConverter/blob/c2328c433c7776e231d86712bb4074c82659e366/Converter/src/chunker_countsort_laszip.cpp#L588)), the `src` and `dst`values are different from `0`.
> 
> Having a look at [`DecoderWorker.js:134`](https://github.com/potree/potree/blob/5636cd471d9eb464969e758be45c44d7613d3859/src/modules/loader/2.0/DecoderWorker.js#L134), the value of the custom attribute is always `0`.
> 
> Happy to help with extra debugging!

I switched to CloudCompare.
